tests/test_backend.py

```diff
@@ -1,26 +1,55 @@
+import os
+import warnings
 from http import HTTPStatus
 from copy import deepcopy
 from unittest import TestCase
 from unittest.mock import patch, MagicMock
 import torch
+from parameterized import parameterized_class
+from transformers import BertForPreTraining, AutoTokenizer
 from torch.fx import GraphModule
 import hidet
 from hidet.graph.frontend.torch.dynamo_backends import get_flow_graph
 from centml.compiler.backend import Runner
 from centml.compiler.server_compilation import CompilationStatus
 from centml.compiler import config_instance
+from .test_helpers import get_graph_module_for_conv_model, get_graph_module_for_llm
 
+resnet_inputs = [torch.zeros(1, 3, 224, 224)]
+os.environ["TOKENIZERS_PARALLELISM"] = "false"
 
+tokenizer = AutoTokenizer.from_pretrained("bert-base-uncased")
+bert_inputs = tokenizer("Hello, my dog is cute", padding='max_length', return_tensors="pt")['input_ids']
+
+model_suite = [
```

Comment:
Instead of defining this as a list, make it a dict. You can use the model name as the key (e.g., "resnet18")